src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/components/page/customheaderlibs.html

```diff
@@ -20,6 +20,7 @@
 #end
 #if ( $includeFormsheadless == "y" )
     <sly data-sly-call="${symbol_dollar}{clientlib.css @ categories='${appId}.forms.react'}" />
+    <sly data-sly-test="${symbol_dollar}{!wcmmode.edit}" data-sly-call="${symbol_dollar}{clientlib.css @ categories='core.forms.components.runtime.all'}"/>
```

Comment:
clientlibs should be included outside of headless flag. if includeForms or includeFormsEnrollment or communication is true.